Adds Python support for the App Dev Guide (#1239)
temporalio/documentation: docs/python/how-to-set-client-options-in-python.md

> ---

Reviewer: Where is this section linked from?

id: how-to-set-client-options-in-python
title: How to set Client options in Python
sidebar_label: Set Client options
description: Set Client options
tags:
  - developer-guide
  - sdk
  - python
---

> To set the Retry Policy, use [`retry_policy`](https://python.temporal.io/temporalio.common.retrypolicy) followed by your Retry statement.

Reviewer: ```suggestion
To set the Retry Policy, set the `retry_policy` in `Client.start_workflow()` or `Client.execute_workflow()` as a [`RetryPolicy`](https://python.temporal.io/temporalio.common.retrypolicy).
```

The following is an example of a Retry Policy.

```python
retry_policy = temporalio.common.RetryPolicy(
    initial_interval=timedelta(milliseconds=1),
    backoff_coefficient=2.0,  # float

>     maximum_interval=time.Second * 100,

Reviewer: This is not valid Python

>     maximum_attempts=0,  # init

Reviewer: What does "init" mean here?

)
```

The previous code example does the following:

- Creates a Retry Policy with initial interval of 1 millisecond.
- Sets the back off coefficient to 2.0.
- Sets the maximum interval of 100 seconds.
- Prevents any retry attempts.

> The following is an example of a Retry Policy in a Workflow Execution.

Reviewer: Not true. It's an example of a retry policy for an activity execution.
